src/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Link,
+} from 'react-router-dom';
+import Books from './components/Books/BookContainer';
+import Category from './components/CategoriesPage/categories';
+import './index.css';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <Router>
+      <nav>
+        <h1>Bookstore CMS</h1>
+        <ul>
+          <li><Link to="/">Books</Link></li>
+          <li><Link to="/CATEGORIES">CATEGORIES </Link></li>
```

Comment:
- [ ] The text **Check Status** should be displayed when clicking on this link. Kindly fix it.